configure

```diff
@@ -322,10 +323,19 @@ cat > "projects/illumos/illumos.sh" <<EOF
 NIGHTLY_OPTIONS="$nopts";			export NIGHTLY_OPTIONS
 GATE="${RELEASE_VER}";				export GATE
 CODEMGR_WS="$conf_root/projects/illumos";	export CODEMGR_WS
+# NOTE: build_illumos modifies this initial constant
 MAX_JOBS=128
 maxjobs() {
         ncpu=\`kstat -p cpu_info:::state | grep -c on-line\`
-        if [[ \$(( \$ncpu + 2 )) -lt \${MAX_JOBS} ]]; then
+	zcpucap=\`pfexec mdata-get sdc:cpu_cap\`
+	if [[ \$zcpucap -eq 0 ]]; then
+		zcpu=\${MAX_JOBS}
+	else
+		zcpu=\`expr \$zcpucap / 100\`
+	vi
```

Comment:
`vi` -> `fi` to close `if` statement

Reply:
Also, since the logic in this function is duplicated in tools/optimize_jobs, why not just call it instead?

Reply:
"vi" --> damn good catch.

"Why not optimize_jobs?" --> Because `nightly.sh` is invoked by $illumos/usr/src/tools/nightly and it has no concept of smartos-live nor any of its subdirectories.  I'd prefer to keep it that way unless there's an existing compelling case of smartos-live awareness I'm missing.

Reply:
Oh, and this part is just creating illumos.sh. That also explains why everything seemed weirdly escaped. 👍